src/videojs-hls.js

```diff
@@ -973,6 +990,20 @@ videojs.Hls.getMediaIndexByTime = function(playlist, time) {
   return -1;
 };
 
+videojs.Hls.prototype.setCurrentTimeByMediaIndex = function(playlist, mediaIndex) {
```

Comment:
Please document this function. Also, we should denote it as private (append an `_`) unless there's a good reason it should be exposed.